docs/io/visualization/how_to_liv_plot.ipynb

```diff
@@ -117,6 +117,32 @@
     "plotter.generate_plot_mpl(packets_mode=\"real\")"
```

Comment:
<p>I think this got left unaddressed from your last PR: it should be "...regroup the bins with broader widths..." - don't we throw warning/error for num_bins &gt; num_shells?</p><p> </p><p>Same for description in mpl cell above</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Yeah it should be "...regroup the bins with broader widths...", I changed the code but forgot to change the docs in the notebook. </p>